desc/objectives/objective_funs.py

```diff
@@ -624,14 +634,44 @@ def _shift_scale(self, x):
         target = (
             self.target / self.normalization if self._normalize_target else self.target
```

Comment:
won't this line fail if bounds were passed and `self._normalize_target = True`? since it would attempt to divide a tuple by a float (it would fail if dividing by any number really)